README.md

```diff
@@ -139,7 +296,63 @@ iex> XDR.String.decode_xdr!(<<0, 0, 0, 17, 84, 104, 101, 32, 108, 105, 116, 116,
 {%XDR.String{max_length: 4294967295, string: "The little prince"}, ""}
 ```
 
-### XDR.Struct
+### Fixed-Length Array
+ 
+Represents a Fixed-Length array which contains the same type of elements
```

Comment:
@FranciscoMolina02 Represents a Fixed-Length array **that** contains the same type of elements